api/src/main/java/com/epam/pipeline/security/saml/SAMLUserDetailsServiceImpl.java

```diff
@@ -62,6 +63,9 @@
     @Value("${saml.user.auto.create: EXPLICIT}")
     private SamlUserRegisterStrategy autoCreateUsers;
 
+    @Value("${saml.user.blocked.attribute: false}")
```

Comment:
Could you please remove `false` from defaults, just leave it empty

Reply:
Done